be/src/vec/CMakeLists.txt

```diff
@@ -21,8 +21,38 @@ set(EXECUTABLE_OUTPUT_PATH "${BUILD_DIR}/src/vec")
 
 file(GLOB_RECURSE VEC_FILES CONFIGURE_DEPENDS *.cpp)
 
+list(FILTER VEC_FILES EXCLUDE REGEX ".*exec/executor/simulator/.*")
+
 add_library(Vec STATIC
         ${VEC_FILES}
 )
 
 pch_reuse(Vec)
+
+if (${BUILD_TASK_EXECUTOR_SIMULATOR} STREQUAL "ON")
```

Comment:
Need a doc for this. Maybe we can add a document in https://doris.apache.org/community/join-community -> Developer Guide

Reply:
ok